driver-sync/src/test/functional/com/mongodb/client/AbstractUnifiedTest.java

```diff
@@ -94,15 +96,22 @@
     private Map<String, BsonDocument> lsidMap;
     private boolean useMultipleMongoses = false;
     private ConnectionString connectionString = null;
-    private final String collectionName = "test";
+    private final String collectionName;
```

Comment:
The new JSON tests for write concern operations specify a collection name that we did not need to check before.